tests/testthat/test_all_open.R

```diff
@@ -64,8 +64,8 @@ if(require(penaltyLearning)){
       target.mat, family="gaussian"))
     expect_is(fit, "iregnet")
     df <- tidydf(fit)
-    ggplot()+
-      geom_point(aes(-log(lambda), weight, color=variable), data=df)
+    #ggplot()+
```

Comment:
why is this commented out?

Reply:
It wasn't really required for the test, didn't affect the test results. Thought they must be there just for trying the code. Un-commented them back